app/models/game.rb

```diff
@@ -42,6 +42,10 @@ def set_white_player_id
     end
   end
 
+  def player_missing?
```

Comment:
Nice work! I really like this method, good thinking. I will make sure to make a story to update the game controller code to use this method once this is merged.